src/chainstate/burn/operations/mod.rs

```diff
@@ -329,6 +335,33 @@ pub struct PegInOp {
     pub burn_header_hash: BurnchainHeaderHash, // hash of the burn chain block header
 }
 
+#[derive(Debug, PartialEq, Clone, Eq, Serialize, Deserialize)]
+pub struct PegOutRequestOp {
+    pub amount: u64,                 // sBTC amount to peg out, in satoshis
+    pub recipient: PoxAddress,       // Address to receive the BTC when the request is fulfilled
+    pub signature: MessageSignature, // Signature from sBTC owner's private key(s)
```

Comment:
To be clear, this is going to be a signature from one of the following:

* A standard principal on Stacks
* A key authorized by the owner of the sBTC (be it a contract or standard principal).

SIP-021 does not speak to the latter, but it should.  It should be possible for the sBTC owner to publish a key that will be used to sign these requests.

Reply:
Alright, I've slightly changed the description and refer to SIP-021 for details. New description:

```
// Signature from sBTC owner as per SIP-021
```